### ISSUE TITLE
DETReg: Unsupervised Pretraining with Region Priors for Object Detection

### ISSUE THREAD

**Issue by Luoyadan** [Luoyadan/Daily-paper-collection#44]
https://arxiv.org/pdf/2106.04550.pdf
### Core Idea
DETReg involves two object-centric and category-agnostic pretraining tasks: an **Object Localization Task** to localize objects, and an **Object Embedding Task** to encode an object’s visual properties

A final object classification head can then be finetuned with a small number of labels yielding better performance than existing methods
![image](https://user-images.githubusercontent.com/17021756/203446813-15e1b8e2-c675-47fd-9207-049c950094bc.png)

#### Object Localization
Selective Search, the region proposal method we adopt in DETReg, uses object cues such as continuity in color, hierarchy, and edges to extract object proposals. DETReg builds upon these region priors to learn a class-agnostic detector during pretraining

#### Object Embedding
💡 aims to predict the embeddings of a separate self-supervised image encoder evaluated on object regions
-> learn embeddings agnostic to transformations

### Summary

Use selective search's output as GT to supervise DETR?
